lib/ansible/modules/packaging/language/pip.py

```diff
@@ -251,7 +251,7 @@ def _is_present(name, version, installed_pkgs, pkg_command):
         else:
             continue
 
-        if pkg_name == name and (version is None or version == pkg_version):
+        if pkg_name == name and (version is None or version == "" or version == pkg_version):
```

Comment:
As mentioned in https://github.com/ansible/ansible/pull/41044#issuecomment-396319765 I'd prefer to set version to None when it is `""` in the main routine instead of duplicating this check to all usage places. This copy-pasting is just asking for bugs to appear in the future.